src/bin/alink.rs

```diff
@@ -212,12 +214,13 @@ fn read_all_objects(args: &Args) -> Result<Vec<Module>, MainError> {
         if path.is_dir() {
             continue;
         }
-        vec.push(read_object(&path, &args)?);
+        vec.push(read_object(&path, args)?);
     }
 
     Ok(vec)
 }
 
+#[allow(clippy::result_large_err)]
```

Comment:
I think it will be better to remove this function-specific macro and add just one at the top of the file like `#![allow(clippy::result_large_err)]`

Reply:
done